examples/ensemble/plot_random_forest_regression_multioutput.py

```diff
@@ -0,0 +1,71 @@
+"""
+============================================================
+Comparing random forests and the multi-output meta estimator
+============================================================
+
+An example to compare multi-output regression with random forest and
+the :ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator.
+
+This example illustrates the use of the
+:ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator
+to perform multi-output regression. A random forest regressor is used,
+which supports multi-output regression natively, so the results can be
+compared.
```

Comment:
Please add some analysis of the output and what it tells us. (I.e. something about the model's failure to exploit correlation hence the dots hanging around the middle...)